packages/openssl.rb

```diff
@@ -1,13 +1,21 @@
 require 'package'
 
 class Openssl < Package
-  version '1.0.1e'
-  binary_url ({
-    i686: 'https://dl.dropboxusercontent.com/s/w6y84tusor5xz5f/openssl-1.0.1e-chromeos-i686.tar.gz?token_hash=AAGQ2xjngbnzme2CKee7Mz5WvkylBtFy1rwUzWDVNuOQ_Q&dl=1',
-    x86_64: 'https://dl.dropboxusercontent.com/s/384awniosicvm12/openssl-1.0.1e-chromeos-x86_64.tar.gz?token_hash=AAH4sdqkNnhIFU-uPdrpqddsi8UU0vWe_gwkplUBM_40MQ&dl=1'
-  })
-  binary_sha1 ({
-    i686: 'cadea32ec770c4b44d565b7e5fdf96a469a05757',
-    x86_64: '3cf4defb11fc2fccce77736d0f4559e56d9d7e05'
-  })
+  version '1.0.2j'
+
+  source_url 'ftp://ftp.openssl.org/source/openssl-1.0.2j.tar.gz' # software source tarball url
+  source_sha1 'bdfbdb416942f666865fa48fe13c2d0e588df54f'          # source tarball sha1 sum
+
+  def self.build                                                  # self.build contains commands needed to build the software from source
+    system './config --prefix=/usr/local'
+    system 'make'                                                 # ordered chronologically
+  end
+
+  def self.install                                                # self.install contains commands needed to install the software on the target system
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"          # remember to include DESTDIR set to CREW_DEST_DIR - needed to keep track of changes made to system
+    system 'cd /usr/local/ssl && \                                # make sure cert.pem exists
+      wget http://curl.haxx.se/ca/cacert.pem && \
+      mv cacert.pem cert.pem'
```

Comment:
`system "wget", "http://curl.haxx.se/ca/cacert.pem", "-O", "#{CREW_DEST_DIR}/usr/local/ssl/cert.pem"`

is better.  If we leave files under CREW_DEST_DIR, crew manges them (correctly removed at crew remove openssl)